### ISSUE TITLE
optimize caliban backend

### ISSUE THREAD

**Issue by ngreenwald** [vanvalenlab/caliban#113]
My computer sounds like it's computing the billionth digit of Pi when running the caliban docker. It requires significantly more system resources than ImageJ or photoshop. Not critical, but something to add to the back burner to investigate.

**Comment by willgraf** [comment]
Can you get an estimate on the amount of RAM it's using?  I am assuming
it's a memory resource issue.

**Comment by ngreenwald** [comment]
I don't see anything with `top`, is there a better way to find out?

**Comment by ngreenwald** [comment]
3.68GB memory, 150% CPU when interacting with labels

**Comment by geneva-miller** [comment]
Zoom commit to master addresses image rendering speed somewhat so I'd like to check back in on this once that update is merged into RGB mode.

**Comment by geneva-miller** [comment]
Did this get any better?

**Comment by ngreenwald** [comment]
It's maybe a bit better, but still pretty noticeable after maybe like 20-30 seconds of use

**Comment by willgraf** [comment]
Any impact on the memory or CPU usage?

Also, it would be good to know which actions (if any) are the biggest culprits here.

**Comment by ngreenwald** [comment]
Brush actions, either drawing or erasing, trigger it the most.
